bc_obps/reporting/migrations/0010_add_reporting_year_data.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.0.7 on 2024-07-31 20:47
+
+from django.db import migrations
```

Comment:
Maybe instead of creating a new migration, we can add it to the existing production data one?

Also no need to delete all existing objects, the table should be empty at that point. Something went wrong if it isn't

Reply:
Modified the PR to run the migrations earlier and removed the deletion with 991c629.